### PR TITLE
Add flask-sspi recipe

### PR DESCRIPTION
**Pull request by fuglede**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment (this one) confirming they are willing to be listed there

### CODE REVIEW

**Comment by fuglede** [comment]
One thing I was a bit unsure about: Since this is a Windows-only package, I figured `noarch: python` didn't apply(?), and so the specification of the `py<36` constraint should happen in the preprocessing selector, not in the `requirements` section; did I understand that correctly?

File: recipes/flask-sspi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-SSPI-{{ version }}.tar.gz
+  sha256: 23440e3b950cb4ab3cf4f5b8079848b9ee1eb6d8d95a2ee34a4d659237439b48
+
+build:
+  skip: True  # [not win or py<36]
```
Comment (chrisburr) [inline recipes/flask-sspi/meta.yaml:14]: This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

Comment (chrisburr) [reply recipes/flask-sspi/meta.yaml:14]: Actually it can't as it can't be build on Linux, sorry about the mix up 😄

Comment (chrisburr) [reply recipes/flask-sspi/meta.yaml:14]: Nominally the fact that this is only makes sense for Windows isn't a problem for `noarch: python` as the Windows-only `pywin32` dependency would prevent it from being installed on Windows.

For conda-forge however, the CI only knows how to build `noarch` packages for Linux (as that's much more common) so this recipe is correct.

Comment (fuglede) [reply recipes/flask-sspi/meta.yaml:14]: Gotcha, thanks for the clarification.

**Pull request merged**